.github/workflows/test-vhd-airgapped.yaml

```diff
@@ -0,0 +1,126 @@
+name: Test VHD Air-gapped

```

Comment:
For consistency, I would use "no egress" instead of "air-gapped"

Reply:
Resolved, changed to no egress